Makefile.defs

```diff
@@ -190,7 +190,7 @@ endif
 
 USE_LUAJIT_LIB=$(or $(DARWIN),$(ANDROID),$(WIN32))
 
-SKIP_LUAJIT_BIN=$(or $(ANDROID),)
+SKIP_LUAJIT_BIN=$(or $(ANDROID),$(MACOS))
```

Comment:
Is this ok?

Reply:
Should be?

Reply:
Shouldn't be?

Reply:
I mean, should be fine, looks like it'd be a value in those cases and otherwise not. But if you're not sure, just test it. ;-)

Reply:
If you no longer call the LuaJIT CLI to launch reader.lua, then, yeah, I guess?

Reply:
Yeah. I'm very bad understanding Makefiles logic. So I do understand that this expression means: in android and macos skip luajit binary. On other platforms don't. If that's correct we're fine :)

Reply:
That's correct, from my very quick glance at it ;).

Reply:
Yes, that's what I meant by "should be?" Should be fine it that's what you mean by it. ;-)